configure.ac

```diff
@@ -106,6 +106,23 @@ AC_FUNC_MMAP
 AC_FUNC_REALLOC
 AC_CHECK_FUNCS([clock_gettime ftruncate memmove memset strdup strstr strtoull strlen strchr])
 
+# Check if __attribute__((constructor)) works. Source:
+# https://github.com/openucx/ucx/blob/72ae40c607067b6dfadf0d208f6811171bbf36b6/config/m4/ucs.m4#L140
```

Comment:
I vote for `AX_GCC_FUNC_ATTRIBUTE` from `https://www.gnu.org/software/autoconf-archive/ax_gcc_func_attribute.html` seem a little more "community" supported